testsuite/podman_runner/12_run_proxy.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
```

Comment:
Maybe to make it clear, we should rename this script and mention that we are also creating the configuration inside it, not only running the container.

Reply:
You are right, I renamed it to `12_setup_and_run_proxy.sh`